### PR TITLE
Create chomikuj.pl.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**
## Type of change

- [x] New Ruleset

### CODE REVIEW

File: src/chrome/content/rules/chomikuj.pl.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		download.box.chomikuj.pl
+		darkorbit.chomikuj.pl
+		deepolis.chomikuj.pl
+		farmerama.chomikuj.pl
+		gangsofcrime.chomikuj.pl
+		gladiators.chomikuj.pl
+		mafia.chomikuj.pl
+		mailgate.chomikuj.pl
+
+	Redirect to HTTP:
```
Comment (J0WI) [inline src/chrome/content/rules/chomikuj.pl.xml:12]: WFM

File: src/chrome/content/rules/chomikuj.pl.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	<target host="docs4.chomikuj.pl" />
+	<target host="docs5.chomikuj.pl" />
+	<target host="docs6.chomikuj.pl" />
+	<target host="docs7.chomikuj.pl" />
+	<target host="docs8.chomikuj.pl" />
+	<target host="docs9.chomikuj.pl" />
+	<target host="mobile.chomikuj.pl" />
+	<target host="simg.chomikuj.pl" />
+
+	<rule from="^http://www\.chomikuj\.pl/" to="https://chomikuj.pl/" />
```
Comment (J0WI) [inline src/chrome/content/rules/chomikuj.pl.xml:56]: [Snapping Redirect](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects)

**Review by J0WI — COMMENTED** [review]
Maybe you also want to add `x[1-4].static-chomikuj.pl`?

**Comment by TotalCaesar659** [comment]
Is it better to create separate file for `x[1-4].static-chomikuj.pl` than add it to this one?

**Comment by J0WI** [comment]
> Added `static-chomikuj.pl` ruleset.

Thanks! Please also add [cross references](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#cross-referencing-rulesets) to each of them.

**Pull request merged**